nulog-inference-service/NulogTrain.py

```diff
@@ -101,20 +88,22 @@ async def consume_signal(job_queue, nw):
 
 
 async def train_model(job_queue, nw):
-    minio_client = boto3.resource(
+    s3_client = boto3.resource(
         "s3",
-        endpoint_url=MINIO_ENDPOINT,
-        aws_access_key_id=MINIO_ACCESS_KEY,
-        aws_secret_access_key=MINIO_SECRET_KEY,
+        endpoint_url=S3_ENDPOINT,
+        aws_access_key_id=S3_ACCESS_KEY,
+        aws_secret_access_key=S3_SECRET_KEY,
         config=Config(signature_version="s3v4"),
     )
-    windows_folder_path = "windows/"
+    windows_folder_path = os.path.join(TRAINING_DATA_PATH, "windows")
     while True:
         new_job = await job_queue.get()  ## TODO: should the metadata being used?
 
-        res_download_data = minio_setup_and_download_data(minio_client)
-        res_train_model = train_nulog_model(minio_client, windows_folder_path)
+        res_s3_setup = s3_setup(s3_client)
+        res_train_model = train_nulog_model(s3_client, windows_folder_path)
         if res_train_model:
+            #Clean windows
+            shutil.rmtree(windows_folder_path)
```

Comment:
@AmartC @dbason  should we remove the window folder? how do we reuse the dumped data, or simply dump from es every training run?